### DISCUSSION TITLE
Abfrageintervall veringern

### DISCUSSION THREAD

**Discussion by SmartMaster2024** [SonnenladenGmbH/APsystems-EZ1-API-HomeAssistant#36] [Q&A]
Hallo zusammen,

ich möchte gerne häufiger die Daten am Wechselrichter abfragen (ca. alle 2 Sekunden). da ein Wechselrichter sehr lange halten soll nun die Frage - verkürzt sich nicht die Lebensdauer wenn der interne Speicher durch häufige Abfragen schneller vollläuft ?

viele Grüße

**Comment by andrewoe** [comment]
... wozu? was soll das bringen?

**Comment by SmartMaster2024** [comment]
weil aktuell alle 10 sek asyn­chron zum meiner Hausverbrauchsermittlung (2sekündlich) läuft und dadurch die Nulleinspeisung nicht gut realisierbar ist.

**Comment by mawoka-myblock** [comment]
Wenn du den WR entfernst und neu einrichtest, dann kannst du den Abfrageintervall selber einstellen.
